src/providerhub/azext_providerhub/generated/_params.py

```diff
@@ -259,22 +326,20 @@ def load_arguments(self, _):
                    id_part='name')
         c.argument('resource_type', type=str,
                    help='The resource type.', id_part='child_name_1')
+        c.argument('nested_resource_type', type=str,
+                   help='The nested resource type.', id_part='child_name_2')
 
     with self.argument_context('providerhub resource-type-registration create') as c:
         c.argument('provider_namespace', type=str,
                    help='The name of the resource provider hosted within ProviderHub.')
         c.argument('resource_type', type=str, help='The resource type.')
-        c.argument('routing_type', arg_type=get_enum_type(['Default', 'ProxyOnly', 'HostBased', 'Extension',
-                                                           'Tenant', 'Fanout', 'LocationBased', 'Failover',
-                                                           'CascadeExtension']), help='The resource routing type.')
+        c.argument('nested_resource_type', type=str,
+                   help='The nested resource type.', id_part='child_name_2')
+        c.argument('routing_type', type=str, help='The resource routing type.')
```

Comment:
It's a breaking change on command line interface.

Reply:
removed nested_resource_type